third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -738,11 +738,17 @@ def arcs_kt_gen(
     schema_name = name + "_schema"
     plan_name = name + "_plan"
 
+    manifest_deps = [
```

Comment:
A comment here would be helpful. The logic in this list comprehension is fairly complex

Reply:
+1